src/Datasource/EntityTrait.php

```diff
@@ -327,9 +327,7 @@ public function &get(string $field): mixed
 
         $method = static::_accessor($field, 'get');
         if ($method) {
-            $result = $this->{$method}($value);
-
-            return $result;
+            return $this->{$method}($value);
```

Comment:
This chance is causing
```
Notice: Only variable references should be returned by reference in /home/runner/work/cakephp/cakephp/src/Datasource/EntityTrait.php on line 330
```

Reply:
Is this correct? Looks weird that this variable needs to be existing just for the return to work^^
I can revert this.